cmd/ocitools/generate.go

```diff
@@ -305,61 +311,81 @@ func setupSpec(g *generate.Generator, context *cli.Context) error {
 		}
 	}
 
-	var sd string
-	var sa, ss []string
+	err := addSeccomp(*g, context)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func addSeccomp(g generate.Generator, context *cli.Context) error {
```

Comment:
And you're following master's existing `setupSpec` pattern here, but I think we may want to flip the argument order (in both this and `setupSpec`) to [put `context` first][1].

[1]: https://blog.golang.org/context#TOC_5.